scripts/add_station.py

```diff
@@ -35,45 +31,46 @@
           'heraringb': [320, 321, 322, 323, 324, 328, 329, 332, 333, 336, 337, 340, 341, 345, 346, 347, 348, 349]}
 
 
-def get_coord_from_file(station_name, coord_filename):
-    coords = None
-    if station_name is not None:
-        fp = open(coord_filename, 'r')
+def Nodes():
```

Comment:
I kind of liked the simple classes here (à la issue #226) but this code is simple enough that it's not a big deal if you prefer a dict-based design. But, having a callable thing with a capitalized name like `Nodes()` that is a function rather than a class violates a virtually universal Python naming convention. This should be renamed to something like `read_nodes()` (and below, `Antennas` → `read_antennas()`).